### ISSUE TITLE
Percolator stats and ES 5.4.1

### ISSUE THREAD

**Issue by piotrkotowski** [sensu-plugins/sensu-plugins-elasticsearch#97]
Hi,
 I am having issues with using:

```
/opt/sensu/embedded/bin# ./ruby metrics-es-cluster.rb -a -p 9201 -h localhost -o
some-host.com.elasticsearch.cluster.status 2 1507793224
some-host.com.elasticsearch.cluster.number_of_nodes 80 1507793224
some-host.com.elasticsearch.cluster.number_of_data_nodes 80 1507793224
some-host.com.elasticsearch.cluster.active_primary_shards 960 1507793224
some-host.com.elasticsearch.cluster.active_shards 1920 1507793224
some-host.com.elasticsearch.cluster.relocating_shards 0 1507793224
some-host.com.elasticsearch.cluster.initializing_shards 0 1507793224
some-host.com.elasticsearch.cluster.unassigned_shards 0 1507793224
some-host.com.elasticsearch.cluster.delayed_unassigned_shards 0 1507793224
some-host.com.elasticsearch.cluster.number_of_pending_tasks 0 1507793224
some-host.com.elasticsearch.cluster.number_of_in_flight_fetch 0 1507793224
some-host.com.elasticsearch.cluster.task_max_waiting_in_queue_millis 0 1507793224
some-host.com.elasticsearch.cluster.active_shards_percent_as_number 100.0 1507793224
Check failed to run: undefined method `[]' for nil:NilClass, ["/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugins-elasticsearch-1.7.1/bin/metrics-es-cluster.rb:170:in `acquire_cluster_metrics'", "/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugins-elasticsearch-1.7.1/bin/metrics-es-cluster.rb:192:in `run'", "/opt/sensu/embedded/lib/ruby/gems/2.4.0/gems/sensu-plugin-1.4.5/lib/sensu-plugin/cli.rb:58:in `block in <class:CLI>'"]
```

Running with sensu 1.1.0-1 and gems as follows:

```
/opt/sensu/embedded/bin# ./gem list | egrep "sensu|elastic"
elasticsearch (1.0.18)
elasticsearch-api (1.0.18)
elasticsearch-transport (1.0.18)
sensu (1.1.0)
sensu-extension (1.5.1)
sensu-extensions (1.9.0)
sensu-extensions-check-dependencies (1.0.1)
sensu-extensions-debug (1.0.0)
sensu-extensions-json (1.0.0)
sensu-extensions-occurrences (1.2.0)
sensu-extensions-only-check-output (1.0.0)
sensu-extensions-ruby-hash (1.0.0)
sensu-json (2.1.0)
sensu-logger (1.2.1)
sensu-plugin (2.0.1, 1.4.5)
sensu-plugins-disk-checks (2.5.0)
sensu-plugins-elasticsearch (1.7.1)
sensu-plugins-environmental-checks (1.1.0)
sensu-plugins-load-checks (4.0.0)
sensu-plugins-memory-checks (3.0.3)
sensu-plugins-network-checks (2.0.1)
sensu-plugins-raid-checks (1.0.0)
sensu-redis (2.2.0)
sensu-settings (10.9.0)
sensu-spawn (2.2.1)
sensu-transport (7.0.2)
```

On Ubuntu 16.04 (if that matters ;) ).

Can you pls let me know is it related to the ES or sensu version (ruby 2.4) that I am using? Do you plan to fix it in near future?

Pls let me know if you need more information.

Thanks a lot!

**Comment by barryorourke** [comment]
I can reproduce the above error using the same version of sensu and the plugin on RHEL.

**Comment by majormoses** [comment]
I would guess that the metrics returned differ on various versions of elasticsearch. I do not have the time to triage this right now but this plugin needs some love as I know it has started getting `bitrot` with regards to version support and someone needs to put in the effort to improve this.

**Comment by eheydrick** [comment]
Fixed in 1.8.1.
